profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -1,300 +1,57 @@
 <?php
-
 /**
  * @file
  * Module file of the translation management Poetry .
  */
 
-/**
- * Implements hook_tmgmt_translator_plugin_info().
- */
-function tmgmt_poetry_tmgmt_translator_plugin_info() {
-  return array(
-    'poetry' => array(
-      'label' => t('Poetry translator'),
-      'description' => t('Poetry Translator service.'),
-      'plugin controller class' => 'TMGMTPoetryTranslatorPluginController',
-      'ui controller class' => 'TMGMTPoetryTranslatorUIController',
-    ),
-  );
-}
+module_load_include('inc', 'tmgmt_poetry', 'inc/tmgmt_poetry.access_control');
 
 /**
- * Local Exception class.
+ * Implements hook_init().
  */
-class TMGMTPoetryException extends Exception {
-  /**
-   * Constructs a TMGMTPoetryException object.
-   */
-  public function __construct($message = '', $data = array(), $code = 0, Exception $previous = NULL) {
-    parent::__construct(strtr($message, $data), $code, $previous);
-  }
-
-}
+function tmgmt_poetry_init() {
 
-/**
- * Implements hook_menu().
- */
-function tmgmt_poetry_menu() {
-  $items = array();
-  $items['tmgmt_poetry/service_callback'] = array(
-    'page callback' => '_tmgmt_poetry_service_callback',
-    'access callback' => TRUE,
-  );
-  return $items;
+  module_load_include('inc', 'tmgmt_poetry', 'inc/tmgmt_poetry.webservice');
 }
```

Comment:
This is pointless. Don't use `hook_init()` to include other files. This will be executed on every request.